Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj

```diff
@@ -1,5 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Target Name="PSRestoreBuiltInModules" AfterTargets="Restore" BeforeTargets="Build">
```

Comment:
maybe a comment?

```
<!-- This target is used to grab the module manifests for ... because they're required for ... -->
```

Reply:
@TylerLeonhardt Could you please add the more proper comment when adding the rest of the module restoring changes? Thanks!